requirements/app/base.txt

```diff
@@ -8,3 +8,4 @@ croniter>=1.3.0, <1.4.0  # strict; TODO: for now until we find something more ro
 traitlets>=5.3.0, <=5.4.0
 arrow>=1.2.0, <1.2.4
 lightning-utilities==0.3.*
+beautifulsoup4==4.8.2
```

Comment:
```suggestion
beautifulsoup4<=4.8.2
```
It's important we don't pin. The pip dependency resolver is not exactly flexible. It is unlikely that for the changes here you require the exact version on the dot, and no other one works.